client/src/components/Navbar/SearchPage.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {useState, useEffect} from 'react';
+import axios from '../../api/Axios';
+import wants from '../../api/Wanted';
+import Cards from '../Slide/Card';
+
+const SearchPage = () => {
+
+    const [query, setQuery] = useState('');
+  const [results, setResults] = useState([]);
+  const [page, setPage] = useState(1);
+  const [showload, setshowload] = useState(false)
+
+  useEffect(() =>{
+    if(query){
+        axios.get(`${wants.getsearch}&query=${query}&page=${page}`)
+        .then(response => {
+            setResults(prevMovies => [...prevMovies, ...response.data.results]);
+        })
+    } else{
+        setResults([])
+    }
+  }, [query, page])
+
+  useEffect(() => {
+    if(results.length > 0){
+        setshowload(true);
+    }else{
+        setshowload(false);
+    }
+  }, [results])
+
+  const InputChange = (e) => {
+    setQuery(e.target.value);
+  }
+
+  return (
+    <div>
+        <br />
```

Comment:
Can you explain why you need to put <br /> tag 6 times.

Reply:
Actually, when this component was initially created then the search bar was rendered on the navbar itself which was not looking good and it was also affecting the functionality of it. So, just to bring the search bar below the navbar on the main page I inserted some breaks from the top.